modules/appeals_api/app/models/appeals_api/notice_of_disagreement.rb

```diff
@@ -345,15 +354,28 @@ def required_claimant_data_is_present
       return if api_version.upcase == 'V1'
 
       # Claimant DOB is always required if they've supplied any claimant headers
-      has_claimant_headers = claimant.birth_date.present?
+      has_claimant_headers = claimant.birth_date.present? && self.class.past?(birth_date) ? true : false
```

Comment:
Check me on this, but I also believe the form headers and data will have already been validated by this point, so if the birthdate hadn't been in the past, it would have already failed. I believe the code can be reverted to the original line.